### PR TITLE
Update vault-user-access-report.py to include groups and their permissions

### PR DESCRIPTION
**Pull request by scottisloud**
This PR updates vault-user-access-report.py to also include groups and their permissions. 

also improves some formatting.

### CODE REVIEW

File: reporting/vault-user-access-report.py [hunk 1]
```diff
@@ -21,19 +23,32 @@ def getAllOwnerVaults():
...
     return vaultList
 
+
 def getSpecifiedVaults():
     with open(inputFilePath, "r", encoding="utf-8") as f:
         vaultList = []
         for id in f:
-            vaultList.append(json.loads(subprocess.run(["op", "vault", "get", id.rstrip(), "--format=json"], check=True, capture_output=True).stdout))
+            vaultList.append(json.loads(subprocess.run(["op", "vault", "get", id.rstrip(
+            ), "--format=json"], check=True, capture_output=True).stdout))
```
Comment (accraw) [inline reporting/vault-user-access-report.py:32]: The closing parenthesis on the newline is a bit weird?

File: reporting/vault-user-access-report.py [hunk 2]
```diff
@@ -21,19 +23,32 @@ def getAllOwnerVaults():
...
 
 # get a list of users and their permissions for a vault
+
+
 def getVaultUserList(vaultID):
     vaultUserList = subprocess.run(
         ["op", "vault", "user", "list", vaultID, "--format=json"], check=True, capture_output=True).stdout
     return vaultUserList
 
+# get all groups in the account
```
Comment (accraw) [inline reporting/vault-user-access-report.py:43]: Why the extra spaces below here?

Comment (scottisloud) [reply reporting/vault-user-access-report.py:43]: VSCode's Python linter is.... opinionated in weird ways (see also your other formatting comment). I don't love some of those decisions either. Let me see if I can make some tweaks to how that behaves......

**Comment by scottisloud** [comment]
Using new formatter that makes a bit more sense.

File: reporting/vault-user-access-report.py (hunk 2 above)
Comment (scottisloud) [reply reporting/vault-user-access-report.py]: just changed to a different auto formatter that's a bit less wonky. Minor improvement over autopep8, which is what was previously used.

**Review by accraw — APPROVED** [review]

**Pull request merged**